package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<package>
+  <name>openface</name>
+  <version>0.0.0</version>
+  <description>The openface package</description>
+
+  <maintainer email="hsr-user@todo.todo">hsr-user</maintainer>
+
+
+  <license>TODO</license>
+
+
+  <url type="website">http://cmusatyalab.github.io/openface/</url>
+
+
+  <!-- Author tags are optional, mutiple are allowed, one per tag -->
+  <!-- Authors do not have to be maintianers, but could be -->
+  <!-- Example: -->
+  <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
+
+
+  <!-- The *_depend tags are used to specify dependencies -->
+  <!-- Dependencies can be catkin packages or system dependencies -->
+  <!-- Examples: -->
+  <!-- Use build_depend for packages you need at compile time: -->
```

Comment:
If you're going to catkinize this package, you should do it completely.  This file needs to include all of the dependencies of the project so we can rosdep install them.